style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
+* Prefer `===` and `!===` to `==` and `!=`.
+* Avoid using arrow functions when not necessary.
```

Comment:
I'm in favor of changing this to always use the fat arrow then. That prevents possible confusion over things not working as expected and it is one less thing to have to worry about.